lightningd/peer_control.c

```diff
@@ -748,6 +748,9 @@ static void json_add_channel(struct lightningd *ld,
 		json_add_node_id(response, "peer_id", &peer->id);
 		json_add_bool(response, "peer_connected", peer->connected == PEER_CONNECTED);
 		json_add_channel_type(response, "channel_type", channel->type);
+		if (channel->ignore_fee_limits) {
+			json_add_bool(response, "ignore_fee_limits", channel->ignore_fee_limits);
+		}
```

Comment:
Did it make sense to make this optional? We add this when the `ingore_fee_limits` only when it is true, why do we not add this also when it is false?

removing the if directly.
```c
json_add_bool(response, "ignore_fee_limits", channel->ignore_fee_limits);
```

usually, as a user, I use the JSON response to understand what I can do, and then I jump inside the docs to see how I can enable it

Reply:
We could do either.  I don't want to promote this, do I hide it unless it's disabled.

In the long run, this should become unnecessary!

Reply:
I see I see